synadm/cli/__init__.py

```diff
@@ -54,6 +63,10 @@ def json_pretty(data):
     return json.dumps(data, indent=4)
 
 
+def json_minify(data):
+    return json.dumps(data, separator=(",", ":"))
```

Comment:
You named a parameter incorrectly, so it doesn't work.

```suggestion
    return json.dumps(data, separators=(",", ":"))
```